app/models/service_template/copy.rb

```diff
@@ -26,4 +32,12 @@ def custom_button_copy(custom_button, template)
   def custom_button_set_copy(custom_button_set, template)
     custom_button_set.deep_copy(:owner => template)
   end
+
+  def resource_action_copy(template)
+    template.resource_actions << resource_actions.collect(&:dup)
```

Comment:
@bdunne Made requested changes and ended up modifying this method a little in hopes of making it more efficient.